ha/cli/exec/clusterExecutor.py

```diff
@@ -129,50 +136,156 @@ def execute(self):
         Log.info("Cluster started successfully")
 
 class ClusterStopExecutor(CommandExecutor):
-    def validate(self) -> str:
+    def validate(self) -> bool:
         raise HAUnimplemented("This operation is not implemented.")
 
-    def execute(self) -> str:
+    def execute(self) -> None:
         raise HAUnimplemented("This operation is not implemented.")
 
 class ClusterRestartExecutor(CommandExecutor):
-    def validate(self) -> str:
+    def validate(self) -> bool:
         raise HAUnimplemented("This operation is not implemented.")
 
-    def execute(self) -> str:
+    def execute(self) -> None:
         raise HAUnimplemented("This operation is not implemented.")
 
 class ClusterStandbyExecutor(CommandExecutor):
-    def validate(self) -> str:
+    def validate(self) -> bool:
         raise HAUnimplemented("This operation is not implemented.")
 
-    def execute(self) -> str:
+    def execute(self) -> None:
         raise HAUnimplemented("This operation is not implemented.")
 
 class ClusterActiveExecutor(CommandExecutor):
-    def validate(self) -> str:
+    def validate(self) -> bool:
         raise HAUnimplemented("This operation is not implemented.")
 
-    def execute(self) -> str:
+    def execute(self) -> None:
         raise HAUnimplemented("This operation is not implemented.")
 
 class ClusterListExecutor(CommandExecutor):
-    def validate(self) -> str:
+    def validate(self) -> bool:
         raise HAUnimplemented("This operation is not implemented.")
 
-    def execute(self) -> str:
+    def execute(self) -> None:
         raise HAUnimplemented("This operation is not implemented.")
 
 class ClusterStatusExecutor(CommandExecutor):
-    def validate(self) -> str:
+    def validate(self) -> bool:
         raise HAUnimplemented("This operation is not implemented.")
 
-    def execute(self) -> str:
+    def execute(self) -> None:
         raise HAUnimplemented("This operation is not implemented.")
 
-class ClusterAddExecutor(CommandExecutor):
-    def validate(self) -> str:
-        raise HAUnimplemented("This operation is not implemented.")
+class ClusterNodeAddExecutor(CommandExecutor):
+    '''
+        Module which will accept the request for cluster add node
+        functinality and which is responsible for delegating that request
```

Comment:
*functionality

Reply:
Done. Thanks for checking this and informing.